sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/ErrorDetails.java

```diff
@@ -0,0 +1,48 @@
+package com.microsoft.azure.mobile.distribute;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Release details JSON schema.
+ */
+class ErrorDetails {
```

Comment:
There is a `message` in payload. It would be better to parse the message and log it.